solutions/sel/lab2/kern/pmap.c

```diff
@@ -187,7 +192,7 @@ mem_init(void)
 	// We might not have 2^32 - KERNBASE bytes of physical memory, but
 	// we just set up the mapping anyway.
 	// Permissions: kernel RW, user NONE
-	// Your code goes here:
+	boot_map_region(kern_pgdir, KERNBASE, (0xffffffff - KERNBASE), 0, PTE_W);
```

Comment:
Could you explain this magic number?

Reply:
Removed the magic number and added an explanation.